src/migrations/1741630574473_scene-stream-access.ts

```diff
@@ -0,0 +1,24 @@
+import { MigrationBuilder, ColumnDefinitions } from 'node-pg-migrate'
+
+export const SceneStreamAccessColumns: ColumnDefinitions = {
+  id: { type: 'uuid', primaryKey: true },
+  place_id: { type: 'text', notNull: true },
+  streaming_url: { type: 'text', notNull: true },
+  streaming_key: { type: 'text', notNull: true },
+  created_at: { type: 'bigint', notNull: true },
+  active: { type: 'boolean', notNull: true, default: true }
```

Comment:
Do we need to keep a record of the tokens that are not being used anymore? What do you think about removing the `active` parameter and just deleting the rows when they can't be used anymore?